### PR TITLE
Change file system permission 

### PR DESCRIPTION
**Pull request by fbugno**
- Change the filesystem permission to host
- ~~patch the desktop file from /usr/bin/retroshare to simple
"retroshare" so the desktop file works on kde.~~
- ~~add the named services to session bus so retroshare can display
a status icon on systray (tested on KDE, in theory freedesktop and
unity named services should cover GNOME and Unity as well).~~

This commit applies to issues #1 and #2

### CODE REVIEW

**Comment by TingPing** [comment]
I'll let the maintainer decide on the filesystem permissions, but if you split the talk permissions and desktop file fix into its own commits/pr they look reasonable to go in.

**Comment by fbugno** [comment]
Ok, done. Now this push request concerns only the file system access permission.

File: cc.retroshare.retroshare-gui.yaml [hunk 1]
```diff
@@ -10,6 +10,7 @@ finish-args:
   - --socket=x11
   - --share=network
   - --persist=.retroshare
```
Comment (TingPing) [inline cc.retroshare.retroshare-gui.yaml]: You would remove this too since it no longer does anything.

Comment (fbugno) [reply cc.retroshare.retroshare-gui.yaml]: Only the persist, right? Ok, removing it.

**Comment by G10h4ck** [comment]
What will happen to profles that where already created with an older version of this package? Will they be accessible automatically or the user will need to copy them in $HOME/.retroshare manually?

A part from this question it seems good to merge to me!

**Comment by TingPing** [comment]
> What will happen to profles that where already created with an older version of this package? Will they be accessible automatically or the user will need to copy them in $HOME/.retroshare manually?

It would have to be manually migrated. (Sorry I didn't notice this last time I glanced)

**Comment by TingPing** [comment]
You could always make a wrapper script that detects the old dir existing and automatically migrates it.

**Comment by TingPing** [comment]
Again to be clear @G10h4ck you just made all users configs get reset.

**Comment by G10h4ck** [comment]
AFAIU @csoler tested it and it worked fine for him, aka without config reset

**Comment by csoler** [comment]
The config reset shouldn't be considered as an issue. The problem is originally that the flatpak installer wasn't correct, so it should be fixed and there's no point in adding more code in the retroshare codebase to make up for this bug. What could be done (if there is a config issue), is to put a word about it in the retroshare.cc Download page in the flatpak section.

**Pull request merged**